### PR TITLE
Create guide_passbolt.rst

### PR DESCRIPTION
**Pull request by thomas-kammerer**

### CODE REVIEW

**Comment by thomas-kammerer** [comment]
"Do not set a passphrase or an expiration date The php-gnupg module does not support using passphrase at the moment. Make sure you do not set a passphrase."
Maybe not the best idea for shared hosting. Perhaps this will be solved soon ;-)

File: source/guide_passbolt.rst [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+.. warning:: Do not set a passphrase or an expiration date.
+
+Save your fingerprint and replace ``SERVER_KEY@EMAIL.TEST`` with your email.
+.. code-block:: console
+ :emphasize-lines: 2,3,4
+
+ [isabell@stardust ~]$ gpg --gen-key
+ [isabell@stardust ~]$ gpg --list-keys --fingerprint
+ [isabell@stardust ~]$ gpg --armor --export-secret-keys SERVER_KEY@EMAIL.TEST > /var/www/virtual/$USER/html/config/gpg/serverkey_private.asc
```
Comment (luto) [inline source/guide_passbolt.rst]: This exposes the private key to the public internet. It seems like only the `webroot` directory should end up in `~/html`. Can you check this and adapt the code?

File: source/guide_passbolt.rst [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+.. warning:: Do not set a passphrase or an expiration date.
+
+Save your fingerprint and replace ``SERVER_KEY@EMAIL.TEST`` with your email.
+.. code-block:: console
+ :emphasize-lines: 4,5
+
+ [isabell@stardust ~]$ mkdir -p passbolt/config
+ [isabell@stardust ~]$ gpg --gen-key
+ [isabell@stardust ~]$ gpg --list-keys --fingerprint
+ [isabell@stardust ~]$ gpg --armor --export-secret-keys SERVER_KEY@EMAIL.TEST > /home/$USER/html/config/gpg/serverkey_private.asc
```
Comment (luto) [inline source/guide_passbolt.rst]: `/home/$USER/html` is a symlink to `/var/www/virtual/$USER/html`, sooo nothing changed here :/

Comment (thomas-kammerer) [reply source/guide_passbolt.rst]: My fault >.<

**Comment by luto** [comment]
Thank you! :tada:

**Pull request closed without merging**